rocAL/include/loaders/video/video_loader.h

```diff
@@ -74,8 +74,8 @@ class VideoLoader : public LoaderModule {
     size_t _sequence_length;
     std::thread _load_thread;
     RocalMemType _mem_type;
-    decoded_image_info _decoded_img_info;
-    decoded_image_info _output_decoded_img_info;
+    decoded_sample_info _decoded_img_info;
```

Comment:
Shouldn't these variables be names as video instead of img?

Reply:
kept it same as there would be too many changes . Now have made this change too